src/boss/boss.go

```diff
@@ -7,11 +7,12 @@ import (
 	"log"
 	"net/http"
 	"os"
-	"strconv"
```

Comment:
Whitespace/formatting improvements are nice, but unless we're doing other changes in the file, let's do these as it's own PR so we don't clutter the logical changes.  You can selectively "git add" files to only include those that matter.